File: source/back-end/src/main/java/be/pxl/backend/service/IDestinationService.java
package be.pxl.backend.service;

import java.util.List;
import be.pxl.backend.entity.Destination;
import org.springframework.stereotype.Service;

public interface IDestinationService {

    public Destination find(int id);

    public List<Destination> all();

    public void persist(Destination destination);

    public void delete(int id);

    public void update(int id, Destination destination);

}

File: Source/frontend/frontend.Service/IDestinationService.cs
using frontend.Domain;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace frontend.Service
{
    public interface IDestinationService
    {
        List<Destination> All();
        Destination Find(int id);
        void Add(Destination destination);
        void Update(Destination destination);
        void Delete(int id);
    }
}
